falco/client.py

```diff
@@ -1,23 +1,46 @@
 import grpc
 
 from falco.client_credentials import get_grpc_channel_credentials
-from falco.domain import Response
-from falco.svc.output_pb2_grpc import serviceStub
+from falco.domain import Response, Request
+from falco.svc.outputs_pb2_grpc import serviceStub
+from falco.svc.version_pb2_grpc import serviceStub as versionServiceStub
+from falco.schema.version_pb2 import request as versionRequest
+from falco.schema.outputs_pb2 import request as outputsRequest
 
 
 class InvalidFormat(Exception):
     pass
 
+class TLSConfigError(Exception):
+    pass
+
+class RequestGenerator:
```

Comment:
I think this should just be a function (and should belong to the outputs domain model)